schema/datatypes/hardware_info.fbs

```diff
@@ -14,6 +14,9 @@ enum McuType: uint16 {
     OWOTRACK_IOS,
     ESP32_C3,
     MOCOPI,
+    WEMOSWROOM02,
+    XIAO_ESP32C3,
```

Comment:
This is mixing board type and mu type. These two are other MCUs that are already here.
```suggestion
```